livekit_room.proto

```diff
@@ -125,3 +128,14 @@ message UpdateSubscriptionsRequest {
 message UpdateSubscriptionsResponse {
   // empty for now
 }
+
+message SendDataRequest {
+  string room = 1;
+  repeated string destination_sids = 2;
```

Comment:
nit: since destination_sids is optional, do you mind placing this last?

Reply:
I made some changes, does it look better?

Reply:
Is the name `destination_sids` ok? or do you have suggestions. 😁